src/git-operations.test.ts

```diff
@@ -112,3 +113,55 @@ describe('didPackageChange', () => {
     expect(execaMock).not.toHaveBeenCalled();
   });
 });
+
+describe('getTags', () => {
+  it('succeeds if repo has complete history and no tags', async () => {
+    execaMock
+      .mockImplementationOnce(async () => {
```

Comment:
:sweat_smile: Took me a moment to realize what was going on here.

Might be better to use a smarter mock implementation here that expects a specific command, so that the tests don't give the wrong answer if the ordering of these calls change or something.